packages/host/app/components/matrix/login.gts

```diff
@@ -210,7 +210,11 @@ export default class Login extends Component<Signature> {
       // component being removed from the DOM and destroyed. Keep in mind that in EC tasks,
       // awaits are really just syntactic sugar for yields, and that we yield to
       // this.matrixService.start()
-      await this.matrixService.start({ auth, refreshRoutes: true });
+      await this.matrixService.start({
+        auth,
+        refreshRoutes: true,
+        loginToAllAccessibleRealmsInBulk: true,
```

Comment:
is there a scenario where we would not want to do this?

Reply:
Actually I can't think of any, I think I can remove this option and make it so that it always logs into all realms that are not logged in when matrix service starts. I'll make the adjustment

Reply:
I removed this option - now there will be a bulk login to realms every time matrix service starts